### ISSUE TITLE
Mappe strade - Luoghi / sentieri raggiungibili a piedi o in bici

### ISSUE THREAD

**Issue by s6carlo** [emergenzeHack/terremotocentro#463]
Non so se esiste un sito con questi dati (stile waze per le strade), ma sarebbe interessante una sezione in cui mostrare i percorsi accessibili a piedi o in bici. Anche questo nel suo piccolo è un modo per riportare gente nei luoghi colpiti dal sisma

**Comment by iltempe** [comment]
ciao @s6carlo conosci la comunità Openstreetmap? www.openstreetmap.org sarebbe lo strumento ideale per fare questo. giusto @alesarrett ?

**Comment by alesarrett** [comment]
Direi proprio di sì. http://www.opencyclemap.org/ per esempio è una "visualizzazione" di Openstreetmap che evidenzia maggiormente i percorsi ciclabili.
Prima però bisognerebbe controllare se ci sono tali informazioni nella mappa (ci sonopiste o percorsi ciclabili in queste zone?) e aggiungere le info sull'accessibilità.

**Comment by s6carlo** [comment]
Si conosco openstreetmap e utilizzo opencyclemap , ma lato utente tramite app. Su opencyclemap ci sono alcuni perscorsi. Sicuramente non è una priorità credo se ne possa riparlare dopo l'inverno.

**Comment by iltempe** [comment]
@s6carlo hai aggiornamenti?

**Comment by s6carlo** [comment]
No, nel senso che non ho trovato delle informazioni aggiornate. Nei luoghi dove le strade sono chiuse (da Visso in poi) non ha molto senso questa mappatura, potrebbe essere utile per tutta l'altra zona. 
Non trovando informazioni in rete ci si dovrebbe basare sulle segnalazioni della gente del posto o chi può andare (con le dovute cautele).
Capisco che non è una priorità, ma credo che con l'arrivo della bella stagione sia utile per riportare qualcuno in questi luoghi. Sinceramente non so nemmeno chi ha la facoltà di "aprire" o meno un sentiero. 
Lo stesso ente parco sibillini si ferma ad un comunicato datato 12/12.
